Integrations/integration-Venafi.yml

```diff
@@ -83,8 +83,19 @@ script:
                 Contents: raw,
                 ContentsFormat: formats.json,
                 ReadableContentsFormat: formats.markdown,
-                EntryContext: {'Venafi.Certificats(val.ID==obj.ID)': certs},
-                HumanReadable: tableToMarkdown('Venafi certificats query response', certs),
+                EntryContext: {'Venafi.Certificates(val.ID==obj.ID)': certs},
+                HumanReadable: tableToMarkdown('Venafi certificates query response', certs),
+            };
+            return entry;
+        case 'venafi-get-details':
+            var res = sendRequest('GET', '/vedsdk/certificates/' + args.guid, undefined, auth.APIKey);
+            entry = {
+                Type: entryTypes.note,
+                Contents: raw,
+                ContentsFormat: formats.json,
+                ReadableContentsFormat: formats.markdown,
+                EntryContext: {'Venafi.Certificate.Details(val.ID==obj.ID)': res},
```

Comment:
This will use the `ID` key in the `Details` object as a unique field, while you want the `ID` key from the `Certificate` object to be the unique field.